### PR TITLE
DB-94 added order-import rule

### PR DESCRIPTION
**Pull request by srmcconomy**
https://roadmunk.atlassian.net/browse/DB-94

### CODE REVIEW

File: lib/order-common.js [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+		reportOutOfOrderNode(context, outOfOrder, found, messageFunc);
+	});
+}
+
+/**
+ * @param {ImportData[]} list
+ * @returns {ImportData[]} modules that are not in the right order
+ */
+function detectOutOfOrder(list) {
+	let maxRank = 0;
```
Comment (idesi) [inline lib/order-common.js]: Align

File: lib/rules/order-require.js [hunk 2]
```diff
@@ -103,11 +31,19 @@ function findVariableDeclaration(node) {
...
 
-function isTopLevelRequire(node) {
+function isTopLevelDeclaration(node) {
 	const declaration = findVariableDeclaration(node);
 	return _.get(declaration, 'parent.type') === 'Program';
 }
 
+function isRequire(node) {
+	return node
+		&& node.callee.type === 'Identifier'
```
Comment (idesi) [inline lib/rules/order-require.js]: A couple of suggestions:

use `isNodeAnIndentifierWithName` from `helper.js`. Consider moving this function to `helper.js`. I am going to work on a rule soon that will need this function as well.

File: docs/rules/order-import.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+import JS from '@roadmunk/jsclass';
+import Moment from 'lib/rm-moment';
+import BaseModel from 'models/BaseModel';
+import Msgbox from 'views/Msgbox';
+
+```
+
+## Things to keep in mind
+- Autofixing logic of this rule currently only fixes one statement at a time.
+- Relative requires at placed at the bottom
```
Comment (alkazar) [inline docs/rules/order-import.md]: "are placed at"

Comment (idesi) [reply docs/rules/order-import.md]: Fixed in https://github.com/Roadmunk/eslint-plugin-roadmunk/pull/14/commits/2d2be98c79cc541c20d6739a93056e6069a6af57

**Pull request merged**